### PR TITLE
FIX: Changes for multi-arch build

### LINKED ISSUE #1473: Devtron is not working on ARM based Kubernetes nodes
If you are trying to resolve an environment-specific issue or have a one-off question about the edge case that does not require a feature then please consider asking a question in Devtron Discord [channel](https://discord.gg/jsRG5qx2gp).

Checklist:

* [yes] I've searched in the docs and FAQ for my answer
* [yes] I've included steps to reproduce the bug.

**Describe the bug**

The Pod and the deployment are not coming in the ready state when trying to deploy the devtron on the ARM node cluster.

**To Reproduce**

Try to deploy the Devtron on the ARM node cluster

**Expected behavior**

Devtron Should work on the ARM as well

**Current behavior** 

POD not coming into Ready state

**Screenshots**
image

**Logs**

standard_init_linux.go:228: exec user process caused: exec format error

AB#286

### PR DESCRIPTION
**Pull request by dheeth**
# Description

Changes required for multi-arch build for devtron with and without CICD mode

Fixes #1473 

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Tested on separate clusters for both ARM and AMD architecture

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas
* [ ] I have tested it for all user roles
* [ ] I have added all the required unit/api test cases

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
 ADD . /go/src/github.com/devtron-labs/devtron/
 RUN GOOS=linux make build-all
 
 # uncomment this post build arg
-FROM alpine:3.15.0 as  devtron-all
-RUN apk add --no-cache ca-certificates
-RUN apk update
-RUN apk add git
-RUN apk add curl
+FROM ubuntu as  devtron-ea
```
Comment (nishant-d) [inline Dockerfile:12]: devtron-all

File: Dockerfile [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+
+RUN apt update
+RUN apt install ca-certificates git curl -y
+RUN apt clean autoclean
```
Comment (nishant-d) [inline Dockerfile:16]: should we use `autoremove` in all dockerfile?

**Review by nishant-d — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (dheeth) [reply Dockerfile]: updated

File: Dockerfile (hunk 2 above)
Comment (dheeth) [reply Dockerfile:16]: already there

**Review by manish-agrawal-ai — APPROVED** [review]

**Review by nishant-d — APPROVED** [review]

**Pull request merged**